Assets/ScriptableObjects/Items/SodiePopperBody.asset

```diff
@@ -15,7 +15,7 @@ MonoBehaviour:
   id: Soda
   displayName: Sodie Popper
   secretName: Diabetic
-  displayDescription: Shake that groove thing, reloads by shakin'.
+  displayDescription: Shake that groove thing, reloads by shakin' your view
```

Comment:
I think we could say "camera" instead of "view" here

Reply:
Agreed (they are robots after all 😔